lua/luasnip/extras/expand_conditions.lua

```diff
@@ -1,13 +1,62 @@
 local M = {}
 
-function M.line_begin(line_to_cursor, matched_trigger)
+-----------------------
+-- CONDITION OBJECTS --
+-----------------------
+local condition_mt = {
+	-- logic operators
+	-- not '-'
+	__unm = function(o1)
+		return function(...)
+			return not o1(...)
+		end
+	end,
+	-- or '+'
```

Comment:
`+` not meaning `and` feels weird to me.

For `or` I think `/` (`__div`) feels more natural: `foo / bar` (foo or bar)
Following this, what about `//` (`__idiv`) for xor?

Reply:
Mhmm `+` for `or` and `*` for `and` are pretty common on the mathematical side of CS, I'd stick to them, fewer surprises. (they do feel weird in the beginning, but one gets used to them :D)
`^` is commonly used for bitwise xor, should keep that one too.